ui/platform_window/x11/x11_window_base.cc

```diff
@@ -273,11 +274,20 @@ void X11WindowBase::SetCapture() {}
 
 void X11WindowBase::ReleaseCapture() {}
 
-void X11WindowBase::ToggleFullscreen() {}
+void X11WindowBase::ToggleFullscreen() {
```

Comment:
"Toggle fullscreen" gives the impression that it does two things:

1) if it is fullscreen , move out of fullscreen

2) if it is not fullscreen, go fullscreen.

this is not the case. it basically  does (2). Maybe

if you do 

SetWMSpecState(!is_fullscreen_, atom_cache_.GetAtom("_NET_WM_STATE_FULLSCREEN"), None);
is_fullscreen_ = !is_fullscreen_;

then you would actually be "toggling the fullscreen" mode. and maybe this method could be called from "::maximize" and "::restore"?